### DISCUSSION TITLE
Natural Inspiration

### DISCUSSION THREAD

**Discussion by kovitikus** [evennia/evennia#2476] [General Discussion]
Similar to how artists use reference images when they draw or paint, I like to use images of nature to inspire me to design mobs or zones. Do you have any cool reference images that you utilize in your creations? Or perhaps something cool you know about and would like to share?

***

Here's a few of the critter inspirations I've saved. I found all of these creatures while browsing Reddit, mostly on the nature subs.
<details>
<summary>

**[Shaman Caterpillar | Royal Moth Caterpillar](https://en.wikipedia.org/wiki/Saturniidae)**
</summary>

![Shaman Caterpillar](https://i.imgur.com/YP6vsHB.jpeg)
</details>

<details>
<summary>

**[Bagworm Moth Caterpillar](https://en.wikipedia.org/wiki/Bagworm_moth)**
</summary>

![Bagworm Moth Caterpillar](https://i.imgur.com/tbPttAc.jpeg)
</details>

<details>
<summary>

**[Roboastra Gracilis](https://en.wikipedia.org/wiki/Roboastra_gracilis)**
</summary>

![Roboastra Gracilis](https://i.imgur.com/gIdt5SB.jpeg)
</details>

<details>
<summary>

**[Armadillo Girdled Lizard](https://en.wikipedia.org/wiki/Armadillo_girdled_lizard)**
</summary>

![Armadillo Girdled Lizard](https://i.imgur.com/lxbQ9qM.png)
</details>

<details>
<summary>

**[Scaly-foot Gastropod](https://en.wikipedia.org/wiki/Scaly-foot_gastropod)**
</summary>

![Scaly-foot Gastropod](https://i.imgur.com/giwVXWa.png)
</details>

<details>
<summary>

**[Sunflower Sea Star](https://en.wikipedia.org/wiki/Sunflower_sea_star)**
</summary>

![Sunflower Sea Star](https://i.imgur.com/O2Xqgcc.jpeg)
</details>

***

I only have one location image that I've stashed away for reference, but I definitely plan on increasing my library when I get around to world building on a serious scale. Right now I'm still hacking away at underlying systems.
<details>
<summary>

**[Eastern Himalayan alpine shrub and meadows in the Barun Valley of Nepal](https://en.wikipedia.org/wiki/Eastern_Himalayan_alpine_shrub_and_meadows)**

</summary>

![Himalayan Meadow](https://i.imgur.com/TphkxoE.jpeg)
</details>

**Comment by Griatch** [comment]
Many of these could certainly work well as exotic 'fantasy' animals.

**Reply by kovitikus** [reply]
Yeah. I was thinking that they don't even have to be used directly as is either.

Like the Bagworm Moth Caterpillar could inspire to me to make some sort of undead beetle-like creature with a tower of bones on it's back. Just some cool design features that can be used in many aspects.
